lib/Elastica/Client.php

```diff
@@ -689,8 +689,6 @@ public function request($path, $method = Request::GET, $data = [], array $query
         } catch (ConnectionException $e) {
             $this->_connectionPool->onFail($connection, $e, $this);
 
-            $this->_log($e);
```

Comment:
shouldn't this be replace by a `$this->_logger->error()`  call ? Overriding the method was deprecated. But now, you removed the logging entirely.

Reply:
@ruflin What you say? I'll update this PR today, I forgot it :sweat_smile:

Reply:
+1 on the change suggested by @stof